### ISSUE TITLE
Webpack unable to handle @mapbox/node-pre-gyp/lib/util/nw-pre-gyp/index.html

### ISSUE THREAD

**Issue by arvindsaraf** [vladmandic/human#423]
**Issue Description**
Migrated from face-api to human, but unable to load the page or do an 'npm run build' in my package due to the following webpack issue. Attaching my package.json - but we are building to run in the browser. Even 'npm run dev' gives the same error when loading the page. 
```
./app/learner/meetings/[meeting_id]/viewer/[viewer_id]/page.tsx
 ⨯ ./node_modules/@tensorflow/tfjs-node/node_modules/@mapbox/node-pre-gyp/lib/util/nw-pre-gyp/index.html
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> <!doctype html>
| <html>
| <head>

Import trace for requested module:
./node_modules/@tensorflow/tfjs-node/node_modules/@mapbox/node-pre-gyp/lib/util/nw-pre-gyp/index.html
./node_modules/@tensorflow/tfjs-node/node_modules/@mapbox/node-pre-gyp/lib/ sync ^\.\/.*$
./node_modules/@tensorflow/tfjs-node/node_modules/@mapbox/node-pre-gyp/lib/node-pre-gyp.js
./node_modules/@tensorflow/tfjs-node/dist/index.js
./node_modules/@vladmandic/human/dist/human.node.js
```

**Steps to Reproduce**

**Expected Behavior**

**Environment**

- Human library version? 3.2.1
- Built-in demo or custom code? Custom code.
- Type of module used (e.g. `js`, `esm`, `esm-nobundle`)?
- TensorFlow/JS version (if not using bundled module)? 
```
    "@tensorflow/tfjs": "^4.17.0",
    "@tensorflow/tfjs-node": "^4.17.0"
```
- Browser or NodeJS and version (e.g. *NodeJS 14.15* or *Chrome 89*)? Chrome 121.0.6167.184, Node v20.11.0
- OS and Hardware platform (e.g. *Windows 10*, *Ubuntu Linux on x64*, *Android 10*)? MacOSX
- Packager (if any) (e.g, *webpack*, *rollup*, *parcel*, *esbuild*, etc.)? 
- Framework (if any) (e.g. *React*, *NextJS*, etc.)? NextJS

**Diagnostics**

- Check out any applicable [diagnostic steps](https://github.com/vladmandic/human/wiki/Diag)

**Additional**

- For installation or startup issues include your `package.json` [package.json](https://github.com/vladmandic/human/files/14399820/package.json)
- 
- For usage issues, it is recommended to post your code as [gist](https://gist.github.com/)
- For general questions, create a [discussion topic](https://github.com/vladmandic/human/discussions)

**Comment by gwendall** [comment]
Solution here: https://github.com/vladmandic/human/issues/418#issuecomment-1936923030

**Comment by vladmandic** [comment]
that's not a temp fix, that is one of possible solutions to what is a general next.js problem - it is a server-side-render framework and it tries to bundle everything/anything server-side. and you want to use human client-side, so you need to tell next.js which flavor of human to use explicitly.

and even if you were trying to use human.js server-side, @tensorflow/tfjs-node must always be excluded from bundler.

closing as not a product issue, but a next.js config issue.

**Comment by arvindsaraf** [comment]
> Temp fix here: [#418 (comment)](https://github.com/vladmandic/human/issues/418#issuecomment-1936923030)

Thanks, this helped unblock
